src/suparnaNJune20/Assignment_31/InputNumberExceptionHandling.java

```diff
@@ -0,0 +1,34 @@
+package suparnaNJune20.Assignment_31;
+import java.util.InputMismatchException;
+import java.util.Scanner;
+public class InputNumberExceptionHandling {
+	static int sum = 0, cnt = 0;
```

Comment:
No need to declare the variables as static.

Reply:
Changes Done.